monkeywagtail/artist/models.py

```diff
@@ -47,14 +47,20 @@ class Artist(ClusterableModel):
     The author snippet gives a way to add authors to a site and create a one-way relationship with content
     """
 
+    title = models.CharField("The artist's name", blank=True, max_length=254)
```

Comment:
Is the change from `artist_name` to `title` just to make it easier to understand, or is there a nuance in using `title` here that is needed by urls.py and views.py?